web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
```

Comment:
Lets update this key to be "review_content", line 29 will also need to be updated.

Reply:
Done